tts/tts.go

```diff
@@ -15,7 +15,7 @@ const (
 	timeout = time.Second * 10
 )
 
-func Create(sentence string, serviceAccountKey []byte, languageCode string) ([]byte, error) {
+func Create(sentence string, serviceAccountKey []byte, languageCode string, voiceName string, speakingRate float32, pitch float32) ([]byte, error) {
```

Comment:
These may need to be `float64` to avoid the need to cast. It is causing the errors below:
```
cannot use speakingRate (type float32) as type float64 in field value
```

Reply:
Update: I changed it to float64.

Reply:
I totally messed that up. I updated everywhere to float32

Reply:
If we go this way we need to cast these values below to float64: `float64(speakingRate)` etc.

Reply:
Got it.

Reply:
Thanks.